### DISCUSSION TITLE
Using <Text /> with chinese text

### DISCUSSION THREAD

**Discussion by abandon888** [pmndrs/drei#1686] [General]
- `three` version: 0.157.0
- `@react-three/fiber` version: 8.14.5
- `@react-three/drei` version: 9.88.2
- `node` version: 16.18.0
- `npm` (or `yarn`) version: 8.19.2

### Problem description:

When insert Chinese text in the Text component import from drei, the text is a messy code.

### Relevant code:

![image](https://github.com/pmndrs/drei/assets/111584427/201ba0f1-8a41-414f-8879-56f3c79362d0)

```js
              <Text
                fontSize={0.5}
                position={[0, 0, 0.5]}
                color="black"
                anchorX="center"
                anchorY="middle">
                请点击盒子
              </Text>
```

**Comment by isaac-mason** [comment]
You might just need to specify a font that supports Chinese characters.

**Comment by isaac-mason** [comment]
See this codesandbox for an example: https://codesandbox.io/s/drei-text-noto-sans-simplified-chinese-2twm7t
